scripts/faucet-deploy

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# shellcheck source=lib/common.sh
+. "${LIB_DIR:-$(cd "${0%/*}/../lib"&&pwd)}/common.sh"
+
+# Deploy Token Faucet (no solc optimization)
+# shellcheck source=/dev/null
+
+dappBuild token-faucet
+if [[ -z "$MCD_GOV" ]]; then
```

Comment:
Why is this check necessary?

Also we should probably move the build step inside the check too right?

Reply:
This was the wrong check, it should be checking whether we already have a FAUCET (i.e. have already exported `$FAUCET`).